sass/blocks/preamble/_preamble.scss

```diff
@@ -4,6 +4,14 @@
   &__inner {
     padding: 4rem 10% 4.2rem;
     text-align: center;
+
+    &--small {
```

Comment:
As we spoke I would drop these modifiers. make it the same as on the form page, or form page same as here